http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/dao/HttpDockerDAO.scala

```diff
@@ -0,0 +1,203 @@
+package org.broadinstitute.dsde.workbench.leonardo
+package dao
```

Comment:
@Qi77Qi taught me this… trying to remember. :)

Reply:
with this, everything under `org.broadinstitute.dsde.workbench.leonardo` namespace will be in scope automatically, so you don't have to `import org.broadinstitute.dsde.workbench.leonardo._`

Reply:
Oh interesting. I can see the appeal in terms of coincidence. At the same time I fear this may make it even easier to set up an implicit hell situation 🙈

Reply:
yea, there needs to be a good reason if we're putting implicits under `org.broadinstitute.dsde.workbench.leonardo`